tldw_Server_API/app/core/DB_Management/db_migration.py

```
@@ -275,22 +275,19 @@ def load_migrations(self) -> list[Migration]:
                 else:
                     continue
             except Exception as e:
-                logger.error(f"Failed to load migration from {filepath}: {e}")
-                continue
+                raise MigrationError(
+                    f"Invalid migration set: {filepath.name}"
+                ) from e
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)